app/models/solidus_avatax_certified/line.rb

```diff
@@ -93,33 +95,37 @@ def refund_lines
     end
 
     def refund_line
-      {
-        number: "#{@refund.id}-RA",
-        itemCode: @refund.transaction_id || 'Refund',
-        quantity: 1,
-        amount: -@refund.amount.to_f,
-        description: 'Refund',
-        taxIncluded: true,
-        addresses: {
-          shipFrom: default_ship_from,
-          shipTo: ship_to
-        }
-      }.merge(base_line_hash)
+      refund_line =
+        {
+          number: "#{@refund.id}-RA",
+          itemCode: @refund.transaction_id || 'Refund',
+          quantity: 1,
+          amount: -@refund.amount.to_f,
+          description: 'Refund',
+          taxIncluded: true,
+          addresses: {
+            shipFrom: default_ship_from,
+            shipTo: ship_to
+          }
+        }.merge(base_line_hash)
+      refund_line.merge(tax_override_hash)
     end
 
     def return_item_line(line_item, quantity, amount)
-      {
-        number: "#{line_item.id}-LI",
-        description: line_item.name[0..255],
-        taxCode: line_item.tax_category.try(:tax_code) || '',
-        itemCode: line_item.variant.sku,
-        quantity: quantity,
-        amount: -amount.to_f,
-        addresses: {
-          shipFrom: get_stock_location(line_item),
-          shipTo: ship_to
-        }
-      }.merge(base_line_hash)
+      return_item_line =
+        {
+          number: "#{line_item.id}-LI",
+          description: line_item.name[0..255],
+          taxCode: line_item.tax_category.try(:tax_code) || '',
+          itemCode: line_item.variant.sku,
+          quantity: quantity,
+          amount: -amount.to_f,
+          addresses: {
+            shipFrom: get_stock_location(line_item),
+            shipTo: ship_to
+          }
+        }.merge(base_line_hash)
+      return_item_line.merge(tax_override_hash)
```

Comment:
For return item here, it will currently add total refund amount (`@refund.amount`) as tax in case of `'full_tax'`.
Should it be just `amount` parameter of this method?